odoo_install.sh

```diff
@@ -377,4 +376,7 @@ echo "Password superadmin (database): $OE_SUPERADMIN"
 echo "Start Odoo service: sudo service $OE_CONFIG start"
 echo "Stop Odoo service: sudo service $OE_CONFIG stop"
 echo "Restart Odoo service: sudo service $OE_CONFIG restart"
+if [ $INSTALL_NGINX = "True" ]; then
+  echo "Nginx configuration at: /etc/nginx/sites-available/odo
```

Comment:
```suggestion
  echo "Nginx configuration at: /etc/nginx/sites-available/odoo
```

Reply:
@Yenthe666  you reviewed the outdated file